content/en/user-guide/tools/transparent-endpoint-injection/index.md

```diff
@@ -85,7 +85,7 @@ For Node.js, you can set this environment variable in your application, to allow
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"
 ```
 
-If you are using the Java AWS SDK v2 in Lambda, you can opt in to automatically disable SSL validation using the configuration `LAMBDA_DISABLE_JAVA_SDK_V2_CERTIFICATE_VALIDATION=1`.
+If you are using the Java AWS SDK v2 in Lambda, LocalStack will per default use bytecode instrumentation to disable certificate validation, so the endpoint injection can work. You can opt out of this behavior by setting `LAMBDA_DISABLE_JAVA_SDK_V2_CERTIFICATE_VALIDATION=0`.
```

Comment:
It would be nice to add one sentence describing the implications of opting out such that the user understands that it can lead to certificate errors when trying to connect to LocalStack using the Java SDK v2.